CMake/Modules/FindNF_NativeAssemblies.cmake

```
@@ -26,6 +26,7 @@ option(API_System.Device.I2c                    "option for System.Device.I2c AP
 option(API_Windows.Devices.Gpio                 "option for Windows.Devices.Gpio API")
 option(API_Windows.Devices.I2c                  "option for Windows.Devices.I2c API")
 option(API_Windows.Devices.Pwm                  "option for Windows.Devices.Pwm API")
+option(API_Windows.Devices.Pwm                  "option for Windows.Devices.Pwm API")
```

guess this should be System.Device.Pwm...
```suggestion
option(API_System.Device.Pwm                  "option for System.Device.Pwm API")
```